src/app/ui-library/inputFeedback.tsx

```diff
@@ -1,23 +1,29 @@
-import { Icon, Info, Warning } from '@phosphor-icons/react';
+import { CheckCircle, Icon, Info, Warning } from '@phosphor-icons/react';
 import styled from 'styled-components';
 import Theme from 'theme';
 import { StyledP } from './common.styled';
 
-export type FeedbackVariant = 'info' | 'danger';
+export type FeedbackVariant = 'info' | 'danger' | 'plain' | 'plainIndented' | 'checkmark';
 
 const colors: Record<FeedbackVariant, string> = {
   info: Theme.colors.white_200,
   danger: Theme.colors.danger_light,
+  plain: Theme.colors.white_200,
```

Comment:
this should be 'explicative' to match zeroheight naming, and white_400
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/bc96faf5-6e9c-46e9-bcbd-1589ed051f11)

figma: 
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/f7386e47-0c2e-4de8-98f7-d397f90027ac)